src/python/WMCore/Storage/StageOutImpl.py

```diff
@@ -79,15 +79,15 @@ def executeCommand(self, command):
 
         """
         try:
-            exitCode = runCommand(command)
-            msg = "%s : Command exited with status: %s\n" % (time.strftime("%Y-%m-%dT%H:%M:%S"), exitCode)
+            exitCode, output = runCommandWithOutput(command)
+            msg = "%s : Command exited with status: %s\n Output message: %s" % (
+                            time.strftime("%Y-%m-%dT%H:%M:%S"), exitCode, output)
             print(msg)
         except Exception as ex:
             raise StageOutError(str(ex), Command=command, ExitCode=60311)
         if exitCode:
-            msg = "%s : Command exited non-zero" % time.strftime("%Y-%m-%dT%H:%M:%S")
-            print("ERROR: Exception During Stage Out:\n")
-            print(msg)
+            msg = "%s : Command exited non-zero: (%s) " % (time.strftime("%Y-%m-%dT%H:%M:%S"), output)
```

Comment:
I'm wondering if we should set `msg` with sort of the same content as above, so providing both exitCode and output. Like "Command exit code non-zero: %s\nOutput message: %s"